src/Symfony/Bridge/Twig/Resources/views/Form/form_table_layout.html.twig

```diff
@@ -42,3 +42,14 @@
         {{- form_rest(form) -}}
     </table>
 {%- endblock form_widget_compound -%}
+
+{%- block empty_row -%}
+    {%- set translation_domain = translation_domain ?: 'sf_form' -%}
```

Comment:
I see now. Shouldn't we rather call the catalog `forms.*.xlf` to be consistent with the other catalogs?